packages/block-editor/src/components/spacing-sizes-control/linked-button.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { link, linkOff } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { Button, Tooltip } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+
+export default function LinkedButton( { isLinked, onClick } ) {
+	const label = isLinked ? __( 'Unlink Sides' ) : __( 'Link Sides' );
+
+	return (
+		<Tooltip text={ label }>
+			<span>
+				<Button
+					className="component-box-control__linked-button"
```

Comment:
```suggestion
					className="component-spacing-sizes-control__linked-button"
```

I'm not sure we should rely on styles from a component in a different package that is exported as experimental. Would it be better to style this explicitly for this component?

Ideally, I'd like to see something like a `LinkedButton` component available in the components package as something similar is used in this, `BoxControl`, `BorderBoxControl`, `BorderRadiusControl` and maybe others. Certainly not something required of this PR of course.

Reply:
Didn't seem to be any styles attached to that class anyway so have removed for now